data/crac-creation/crac-creator-csa-profiles/src/test/java/com/farao_community/farao/data/crac_creation/creator/csa_profile/crac_creator/remedial_action/InjectionSetPointActionCreationTest.java

```diff
@@ -129,4 +130,41 @@ void testIgnoreInvalidInjectionSetpointProfile() {
         CsaProfileCracCreationTestUtil.assertRaNotImported(cracCreationContext, "parent-remedial-action-17", ImportStatus.INCONSISTENCY_IN_DATA, "Remedial action parent-remedial-action-17 will not be imported because there is no StaticPropertyRange linked to that RA");
         CsaProfileCracCreationTestUtil.assertRaNotImported(cracCreationContext, "parent-remedial-action-18", ImportStatus.INCONSISTENCY_IN_DATA, "Remedial action parent-remedial-action-18 will not be imported because several conflictual StaticPropertyRanges are linked to that RA's RotatingMachineAction");
     }
+
+    @Test
+    void testImportShuntCompensatorModifications() {
```

Comment:
There should also be 6 not imported shunt compensator actions. Could you check, for each, why it was not imported with the `assertRaNotImported` please?